src/qt/utilitydialog.cpp

```diff
@@ -29,18 +31,17 @@
 #include <QVBoxLayout>
 
 /** "Help message" or "About" dialog box */
-HelpMessageDialog::HelpMessageDialog(QWidget *parent, bool about) :
-    QDialog(parent, GUIUtil::dialog_flags),
-    ui(new Ui::HelpMessageDialog)
+HelpMessageDialog::HelpMessageDialog(QWidget* parent, bool about, const NetworkStyle* networkStyle) : QDialog(parent, GUIUtil::dialog_flags),
+                                                                                                      ui(new Ui::HelpMessageDialog)
```

Comment:
Break the long line for better readability:
```suggestion
HelpMessageDialog::HelpMessageDialog(QWidget* parent, bool about, const NetworkStyle* networkStyle)
    : QDialog(parent, GUIUtil::dialog_flags),
      ui(new Ui::HelpMessageDialog)
```
?

Reply:
Ok, will do.